files/build/scripts/buildinfo_base.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+BUILDINFO_PATH=/usr/local/share/buildinfo
+
+LOG_PATH=$BUILDINFO_PATH/log
+VERSION_PATH=$BUILDINFO_PATH/versions
+PRE_VERSION_PATH=$BUILDINFO_PATH/pre-versions
+DIFF_VERSION_PATH=$BUILDINFO_PATH/diff-versions
+BUILD_VERSION_PATH=$BUILDINFO_PATH/build-versions
+POST_VERSION_PATH=$BUILDINFO_PATH/post-versions
+VERSION_DEB_PREFERENCE=$BUILDINFO_PATH/versions/01-versions-deb
+
+. $BUILDINFO_PATH/config/buildinfo.config
+
+URL_PREFIX=$(echo "${PACKAGE_URL_PREFIX}" | sed -E "s#(//[^/]*/).*#\1#")
+
+log_err()
+{
+    echo "$1" >> $LOG_PATH/error.log
+    echo "$1" 1>&2
+}
+
+check_version_control()
+{
+    if [[ ",$SONIC_VERSION_CONTROL_COMPONENTS," == *,all,* ]] || [[ ",$SONIC_VERSION_CONTROL_COMPONENTS," == *,$1,* ]]; then
+        echo "y"
+    else
+        echo "n"
+    fi
+}
+
+ENABLE_VERSION_CONTROL_DEB=$(check_version_control "deb")
+ENABLE_VERSION_CONTROL_PY2=$(check_version_control "py2")
+ENABLE_VERSION_CONTROL_PY3=$(check_version_control "py3")
+ENABLE_VERSION_CONTROL_WEB=$(check_version_control "url")
```

Comment:
>WEB [](start = 23, length = 3)

better name: ENABLE_VERSION_CONTROL_URL

Reply:
I changed back to use web, the same as in the document, how about not use url?